src/cgmemmgr.cpp

```diff
@@ -506,13 +506,15 @@ class ROAllocator {
     virtual ~ROAllocator() {}
     virtual void finalize()
     {
+#ifndef _CPU_AARCH64_
```

Comment:
As mentioned in the comment, this is certainly wrong, if you need this for testing, at least enable it for only freebsd.